support-frontend/assets/pages/[countryGroupId]/components/thankyou.tsx

```diff
@@ -177,9 +178,16 @@ export function ThankYouComponent({
 	const isNewAccount = userTypeFromIdentityResponse === 'new';
 	const emailExists = !isNewAccount && isSignedIn;
 
+	const abParticipations = abTestInit({ countryId, countryGroupId });
+	const showNewspaperArchiveBenefit = ['v1', 'v2', 'control'].includes(
+		abParticipations.newspaperArchiveBenefit ?? '',
```

Comment:
Is the nullish coalescing required here? My guess would be that TS is okay with `['v1', 'v2', 'control'].includes(undefined)` but I might be missing something!

Reply:
Hope ok but I get from the includes using undefined ->  `Argument of type 'string | undefined' is not assignable to parameter of type 'string'. Type 'undefined' is not assignable to type 'string'.ts(2345)`

Reply:
👍  Yep all good, sorry for the derail!